### PR TITLE
Minor fix params description for addPercent function

### PR DESCRIPTION
**Pull request by bee0060**
Just fix function's comment

### CODE REVIEW

File: dangerfile.js [hunk 1]
```diff
@@ -53,7 +53,8 @@ function generateMDTable(headers, body) {
 
 /**
  * Generates a user-readable string from a percentage change
- * @param {string[]} headers
+ * @param {number} change
+ * @param {boolean} includeEmoji
  */
 function addPercent(change, includeEmoji) {
   if (!isFinite(change)) {
```
Comment (bee0060) [inline dangerfile.js]: I just reviewed some code. And found code on line 65
```javascript
if (/^-|^0(?:\.0+)$/.test(formatted)) {
```

I am not sure the purpose of this code, but when formatted equals `-.2abc` this `.test(formatted)` will also return true, does it meet the expectation ? Or is it a bug?

Comment (aweary) [reply dangerfile.js]: `formatted` should never be an arbitrary string, so cases like `-2.abc` aren't really a concern.

**Comment by aweary** [comment]
We generally avoid merging changes that don't include any actual features/bug fixes, but since these JSDoc headers can confuse contributors this change seems fine. Thanks!

**Comment by bee0060** [comment]
Thanks!
Yeah, this PR just a minor change of comment. I just started to read react's code. Hope I could contribute some actual features or bug fixing later.

**Pull request merged**